edge-util/src/Microsoft.Azure.Devices.Edge.Util/azureLogAnalytics/AzureLogAnalytics.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util.AzureLogAnalytics
+{
+    using System;
+    using System.IO;
+    using System.Net;
+    using System.Security.Cryptography;
+    using System.Text;
+
+    /* Sample code from:
+    /* https://github.com/veyalla/MetricsCollector/blob/master/modules/MetricsCollector/AzureLogAnalytics.cs
+    /* https://dejanstojanovic.net/aspnet/2018/february/send-data-to-azure-log-analytics-from-c-code/
+    */
+
+    public class AzureLogAnalytics
+    {
+        public AzureLogAnalytics(string workspaceId, string sharedKey, string logType, string apiVersion = "2016-04-01")
+        {
+            this.WorkspaceId = workspaceId;
+            this.SharedKey = sharedKey;
+            this.LogType = logType;
+            this.ApiVersion = apiVersion;
+        }
+
+        public string WorkspaceId { get; }
+        public string SharedKey { get; }
+        public string ApiVersion { get; }
+        public string LogType { get; }
+
+        public void Post(byte[] content)
```

Comment:
I believe this method is specific to write data to log table, not metric or others; so I will suggest to name it as "PostLog" or "PostToLog" etc.

Reply:
nit: should it be more friendly to support content as string type and convert it to UTF8 encoding?

Reply:
The UTF8 format is entirely just to be consistent with what Venkat is doing for his Log Analytic. I created ` PostAsync() ` that uses the HttpClient lib which takes JSON as input.